src/bp/nlu-core/initialize-tools.ts

```diff
@@ -43,19 +40,18 @@ const initializeLanguageProvider = async (bp: typeof sdk) => {
   }
 }
 
-const initDucklingExtractor = async (bp: typeof sdk): Promise<void> => {
-  const globalConfig = (await bp.config.getModuleConfig('nlu')) as Config
-  await DucklingEntityExtractor.configure(globalConfig.ducklingEnabled, globalConfig.ducklingURL, bp.logger)
+const initDucklingExtractor = async (config: NLU.Config, logger: NLU.Logger): Promise<void> => {
+  await DucklingEntityExtractor.configure(config.ducklingEnabled, config.ducklingURL, logger)
 }
 
-export async function initializeTools(bp: typeof sdk): Promise<Tools> {
-  await initDucklingExtractor(bp)
-  const { languageProvider } = await initializeLanguageProvider(bp)
-  const { MLToolkit: mlToolkit, logger } = bp
+export async function initializeTools(config: NLU.Config, logger: NLU.Logger): Promise<Tools> {
+  await initDucklingExtractor(config, logger)
+  const { languageProvider } = await initializeLanguageProvider(config, logger)
 
   return {
     partOfSpeechUtterances: (tokenUtterances: string[][], lang: string) => {
-      const tagger = getPOSTagger(lang, mlToolkit)
+      const tagger = getPOSTagger(lang, MLToolkit)
+      // @ts-ignore
```

Comment:
why ts-ignore ? because of .bind ? then change the line to 

```ts
tokenUtterances.map(u => tagSentence(tagger, u))
```

Reply:
sure